modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryImpl.java

```diff
@@ -419,6 +447,32 @@ protected boolean checkAckTimeout(long ackTimeout) {
         return true;
     }
 
+    /** */
+    public void processMsg(TcpDiscoveryMetricsUpdateMessage msg, long tsNanos) {
```

Comment:
This is too common name for a method that does such a specific thing. Please rename it.